packages/teams-js/src/public/stageView.ts

```diff
@@ -0,0 +1,64 @@
+import { sendMessageToParent } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { SdkError } from '../public';
+import { FrameContexts } from './constants';
+
+/**
+ * Namespace to interact with the stage view specific part of the SDK.
+ */
+export namespace stageView {
+  /**
+   * Parameters to open a stage view.
+   */
+  export interface StageViewParams {
+    /**
+     * The application ID of the Teams application to be opened.
+     */
+    appId: string;
+
+    /**
+     * The URL of the content to display.
+     */
+    contentUrl: string;
+
+    /**
+     * The chat or channel ID.
+     */
+    threadId: string;
+
+    /**
+     * The title to give the stage view.
+     */
+    title: string;
+
+    /**
+     * The Teams app website URL.
+     */
+    websiteUrl?: string;
+
+    /**
+     * The entity ID of the Teams app content being opened.
+     */
+    entityId?: string;
+  }
+
+  /**
+   * @private
```

Comment:
I know this is still a draft so no rush, but I thought it would be easier to let you know sooner than later: we have to change the `@private` tags from v1 to `@hidden` in v2 because we use a different docs generator and that's the tag that will correctly hide the docs as intended. (This wasn't anywhere in the git magician guide yet I think, so totally not something you could have known. I'm adding to a list of things to add to the wiki guide and these will be good additions!)